universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -54,14 +56,21 @@ public QuestionController(
         .getReadOnlyQuestionService()
         .thenApplyAsync(
             readOnlyService -> {
+              String exception = "";
               try {
                 QuestionDefinition definition = questionForm.getBuilder().setVersion(1L).build();
-                service.create(definition);
+                boolean success = service.create(definition).isPresent();
+                if (!success) {
+                  exception =
+                      String.format(
+                          "create failed: this is most likely you specify an invalid path %s",
+                          definition.getPath());
+                }
               } catch (UnsupportedQuestionTypeException e) {
-                // I'm not sure why this would happen here, so we'll just log and redirect.
+                exception = e.toString();
                 LOG.info(e.toString());
```

Comment:
No sense in calling `e.toString()` twice. Just use `LOG.info(exception)`

Reply:
Done.